flake.lock

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Please remove these flake files. I don't use nix and have no way of maintaining them.